src/js/edu-benefits/containers/IntroductionPage.jsx

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 
 class IntroductionPage extends React.Component {
+  componentDidMount() {
+    const breadcrumbs = document.querySelector('.va-nav-breadcrumbs-list');
+    breadcrumbs.setAttribute('tabindex', '-1');
```

Comment:
We could. I was assuming they exist here because they always should, whereas the other header elements are different depending on the page.